src/app/summary/summary.component.ts

```diff
@@ -30,10 +31,16 @@ export class SummaryComponent implements OnInit {
     console.log('landing page');
   }
 
+=======
+>>>>>>> Fixed Linter Errors so not it runs.
```

Comment:
Some more of those merge conflict messages being added I think

Reply:
Yeah this needs to be gone, I'm surprised it compiles.